irohad/ametsuchi/impl/postgres_query_executor.cpp

```diff
@@ -16,7 +16,9 @@
 #include <boost/range/algorithm/transform.hpp>
 #include <boost/range/irange.hpp>
 #include "ametsuchi/impl/soci_utils.hpp"
+#include "backend/protobuf/permissions.hpp"
 #include "common/types.hpp"
+#include "cryptography/public_key.hpp"
```

Comment:
Is this include needed?

Reply:
Yes, it's used in line 411:
`                    return shared_model::interface::types::PubkeyType{
                        shared_model::crypto::Blob::fromHexString(public_key)};`